### PR TITLE
Minor upgrades to react router 4 example

### PR DESCRIPTION
**Pull request by rek**

### CODE REVIEW

File: examples/react-router-4/components/App.js [hunk 1]
```diff
@@ -28,9 +35,10 @@ function App({ logout }) {
           {' '}
           <Link to="/admin">{'Admin'}</Link>
           {' '}
-          <Link to="/login">Login</Link>
-          {' '}
-          <button onClick={() => logout()}>Logout</button>
+          {user.data ?
+            <button onClick={() => logout()}>Logout</button> :
+            <Link to="/login">Login</Link>
+          }
```
Comment (mjrussell) [inline examples/react-router-4/components/App.js]: I like the idea of hiding the login/logout buttons. Maybe the examples should use the `userAuthWrapper` hoc? The ternary is probably cleaner but then the example can show more of the API

Comment (rek) [inline examples/react-router-4/components/App.js]: To be honest, I only found this project last night and have not had much of a chance to look at all the API, I agree examples are a good chance to show that stuff, I'll take a look and see what I can manage here.

**Comment by mjrussell** [comment]
No problem! I can also throw up what Im talking about real quick, I just at least wanted to explain why I might do things a little differently

**Comment by rek** [comment]
Awesome! Would love to see it :)

**Comment by mjrussell** [comment]
Maybe something like this:
https://github.com/mjrussell/redux-auth-wrapper/commit/8e404a77a784e6e39a75be0372614aaa4c2b644b

Definitely a few different ways to do it

**Comment by rek** [comment]
Doing it that way is much nicer!

(Though I still prefer to map over the call's in app.js)

I'll close this so you can merge in your more awesome work.

**Comment by mjrussell** [comment]
OK great! I'll merge it then.

I agree I like your use of map coming from an FP background but worried it might throw some people not as used to thinking about it that way in the initial example 😄

**Comment by rek** [comment]
Ahh again, good point!

**Pull request closed without merging**